Микросетки на флексах для всех страниц

Nemchenco96 opened the pull request:
---
:mortar_board: [Микросетки на флексах для всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2065035/tasks/11)

:boom: https://htmlacademy-htmlcss.github.io/2065035-sedona-35/5/

Nemchenco96 committed:
Тема 6

Dima-Dim commented on styles/styles.css:
```diff
@@ -50,29 +50,50 @@ body {
 .page-header {
   display: flex;
   width: 1200px;
+  height: 64px;
```
Не забудь убрать высоты у всех блоков в которых так или иначе есть текст.

Dima-Dim commented on styles/styles.css:
```diff
@@ -50,29 +50,50 @@ body {
   width: 1200px;
+  height: 64px;
   margin: 0 auto;
+  max-height: 64px;
```
Прям чтобы точно из него всё вывалилось если вдруг станет больше информации :)

Dima-Dim commented on styles/styles.css:
```diff
@@ -50,29 +50,50 @@ body {
 .header-logo {
-  width: 140px;
+  width: 138px;
+  height: 70.05px;
```
Точности до целых будет более чем достаточно.

Dima-Dim commented on styles/styles.css:
```diff
@@ -148,6 +176,34 @@ body {
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  margin-top: 92px;
```
На самом деле это padding у родительского блока.

Dima-Dim commented on styles/styles.css:
```diff
@@ -148,6 +176,34 @@ body {
+  margin-left: 85px;
+  margin-right: 85px;
+  margin-bottom: 0;
+  margin-top: 31px;
```
Это margin у предыдущего блока.

Dima-Dim commented on styles/styles.css:
```diff
@@ -148,6 +176,34 @@ body {
+
+.advantages-description {
+  display: block;
+  margin-left: 85px;
```
А тут должно быть выравнивание по центру, а не "забивание гвоздей". Есть ощущение что материал перед выполнением домашнего задание проработан не очень хорошо.

Dima-Dim reviewed (changes requested):
Пересмотреть материал, в частности демки и постараться проследить логику того какие размеры для чего и как задаются.

Nemchenco96 committed:
Исправление. Тема-6

1) Высоту у блоков с текстом убрал.
2) Выравнивание по центру выполнил.
3) Доработал классы, о чем говорили на консультации.
4) Поменял в блоках внешние отступы на pdding у родительских элементов.

Dima-Dim commented on catalog.html:
```diff
@@ -221,13 +222,13 @@ <h4 class="hotels-card-title">GreenTree Inn</h4>
+           <input class="field-catalog"  placeholder="Ваш e-mail" type="email" name="newsletter-email" id="newsletter-email" required>
             <button class="button newsletter-button" type="submit">
-              <span>Подписаться</span>
+              <span class="button-text">Подписаться</span>
```
Для чего `span`?

Dima-Dim commented on styles/styles.css:
```diff
@@ -277,89 +446,193 @@ body {
+
 .button-apply {
+  width: 191px;
+  height: 36px;
```
Приведёт к проблемам при переполнении.

Dima-Dim commented on styles/styles.css:
```diff
@@ -383,19 +659,81 @@ body {
+  box-sizing: border-box;
+  width: 60px;
+  height: 60px;
+  padding: 23.45px 11.36px 24.55px 12.64px;
```
Точности до целых достаточно.

Dima-Dim reviewed (commented):
[images/list.svg](https://github.com/htmlacademy-htmlcss/2065035-sedona-35/pull/5/files#diff-c347c21fd78e1d3d2d7f533ddc7c681ce735aadae35a2017d3106e6420c8295e) - что это? :) почему не css?
